src/python/pants/backend/build_files/fmt/black/register.py

```diff
@@ -16,7 +17,7 @@ class BlackRequest(FmtBuildFilesRequest):
 
 @rule(desc="Format with Black", level=LogLevel.DEBUG)
 async def black_fmt(request: BlackRequest.Batch, black: Black) -> FmtResult:
-    black_ics = await Black._find_python_interpreter_constraints_from_lockfile(black)
+    black_ics = await get_lockfile_interpreter_constraints(black)
```

Comment:
It made no sense for this to be a staticmethod on the PythonToolBase subclass, as we had to pass the instance in anyway.